### ISSUE TITLE
Why don't make NewPipe look like YouTube - Vanced.Pro 

### ISSUE THREAD

**Issue by huuson889** [TeamNewPipe/NewPipe#8266]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

I have seen an app in [Vanced.pro](https://www.vanced.pro/) , I think it use NewPipe source codes but Vanced.pro is very friendly user interface, just look like YouTube, but just one think I disappointed is Vanced.pro have its own ads
It will be great if NewPipe make something like Vanced.pro

**Comment by SameenAhnaf** [comment]
Looks like a duplicate of https://github.com/TeamNewPipe/NewPipe/issues/6395. Scroll the comments to make sure if that is what you want.

**Comment by huuson889** [comment]
No. It is completely different, what I said is Vanced.pro not PureTube in PlayStore. Go to https://www.vanced.pro/ the install the app in that, you will understand what I said

**Comment by tsiflimagas** [comment]
No, it's not completely different. Vanced.pro looks like another copy of Newpipe, which seems to be violating it's license, like the case seems to be with PureTube. The only difference is that you generically ask for the whole UI to change, while #6395 asks just for the homepage.

**Comment by huuson889** [comment]
Okay I've just read #6395 but anyway, Vanced.pro is very nice, It loads video faster than NP, Idk why that can happen, and also it almost exactly same like YT,  I have used PureTube before but I remembered it must pay something to use, Vanced.pro is completely free but just ads, I hate that.
So, **just focus the main point:** **Make NewPipe like Vanced.pro please, I think it's possible, Vanced.pro can do it, so can NewPipe, right?**

**Comment by mleithner** [comment]
This issue lacks details. Judging from the site you linked, it looks virtually identical to NewPipe (I mean, it illegally uses its code...), so you will have to specify precisely which changes you would like to see, including screenshots and descriptions of behavior.

Besides, I don't see any reason for big changes to NewPipe's UI. It seems more functional than that of the official YT app to me.

**Comment by Stypox** [comment]
NewPipe is not a copy of the official YouTube app, it is an app of its own. This is clear since e.g. NewPipe does not only support YouTube, and also because some things behave intentionally differently (e.g. the background player does not exist on YouTube). You could argue that the UI is not perfect and you would be right, like our issue tracker confirms. But the way people use NewPipe is different than how they use YouTube, since they are two different apps. Hence transforming the navigation/workflow of NewPipe into the one of YouTube would just make many users angry and confused. This already happened before with the unified player and the bottom mini-player. Also, I don't see specific advantages to reworking everything, other than lowering a bit the effort people need to switch from YouTube to NewPipe. NewPipe's UI is not deeply flawed, so how it looks in the end (if YouTube-like or not) is just a matter of personal preference, so the NewPipe UI will not be changed drastically as you wish.

The license allows you to create a fork and change the UI to what you wish (though, as others said, Vanced-pro violates that licence because their code is closed-source and without attribution).
